src/Appwrite/Platform/Workers/Certificates.php

```diff
@@ -126,7 +126,7 @@ private function execute(Domain $domain, Database $dbForConsole, Mail $queueForM
         $certificate = $dbForConsole->findOne('certificates', [Query::equal('domain', [$domain->get()])]);
 
         // If we don't have certificate for domain yet, let's create new document. At the end we save it
-        if (!$certificate) {
+        if (empty($certificate) || $certificate->isEmpty()) {
```

Comment:
`$certificate` will always be a `Document`, right? So, we just need the `$certificate->isEmpty()` check?